react/src/components/VFolderSelect.tsx

```diff
@@ -96,14 +98,17 @@ const VFolderSelect: React.FC<VFolderSelectProps> = ({
   // TODO: use controllable value
```

Comment:
Let's remove this line.

Reply:
I modified it ! https://github.com/lablup/backend.ai-webui/pull/2572/files#diff-d191efffefe30deea0936aba97b2645040e05e1276691079ca15866550abbdefL96